pyproject.toml

```diff
@@ -332,7 +332,8 @@ exclude_also = [
    'class .*\bProtocol\b.*\):',
    "raise NotImplementedError",
    '.*if "sphinx" in sys.modules:',
-   'TODO: test this line'
+   'TODO: test this line',
+   'if __name__ == "__main__":',
```

Comment:
if adding this you can remove the `pragma: no cover` from the other two instances  of `if __name__ == "__main__"` in the code